sjsonnet/test/src-jvm-native/sjsonnet/ErrorTests.scala

```diff
@@ -289,28 +282,40 @@ object ErrorTests extends TestSuite{
       )
     }
 
-    test("import_wrong_nr_args") - checkImports(
+    test("import_wrong_nr_args") - check(
       """|sjsonnet.Error: Function parameter y not bound in call
          |    at [Apply1].(sjsonnet/test/resources/imports/error.import_wrong_nr_args.jsonnet:3:6)
-         |""".stripMargin
+         |""".stripMargin,
+      suite = "imports"
     )
 
-    test("wrong_named_arg") - checkImports(
+    test("wrong_named_arg") - check(
       """|sjsonnet.Error: Function has no parameter z
          |    at [Apply].(sjsonnet/test/resources/imports/error.wrong_named_arg.jsonnet:3:6)
-         |""".stripMargin
+         |""".stripMargin,
+      suite = "imports"
     )
 
-    test("too_many_arg") - checkImports(
+    test("too_many_arg") - check(
       """|sjsonnet.Error: Too many args, function has 2 parameter(s)
          |    at [Apply].(sjsonnet/test/resources/imports/error.too_many_arg.jsonnet:3:6)
-         |""".stripMargin
+         |""".stripMargin,
+      suite = "imports"
     )
 
-    test("too_many_arg_with_named_arg") - checkImports(
+    test("too_many_arg_with_named_arg") - check(
       """|sjsonnet.Error: binding parameter a second time: x
          |    at [Apply].(sjsonnet/test/resources/imports/error.too_many_arg_with_named_arg.jsonnet:2:2)
-         |""".stripMargin
+         |""".stripMargin,
+      suite = "imports"
+    )
+
+    test("format_func") - check(
+      """|sjsonnet.Error: Cannot format function value
+         |    at [Function].(sjsonnet/test/resources/db/error.format_func.jsonnet:1:9)
```

Comment:
...and unlike the C++ implementation we even produce a useful position that points the the failing argument